src/style.css

```diff
@@ -62,6 +62,9 @@
   --color-heading: #1a202c;
   --color-heading-black: black;
   --color-accent: #d1dce5;
+  --color-white: #f7fafc;
```

Comment:
Maybe let's give this color a better name than `white`? It's always good for `#fff` to remain "white" in your code.